src/component/index.tsx

```diff
@@ -1,55 +1,76 @@
 import * as React from 'react';
```

Comment:
taking advantage of Node and Webpack's index-require convention for convenience. i think it also clarifies to anyone looking at this directory which file is the root.